Index: PythonProject/web_task/functions.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/PythonProject/web_task/functions.py b/PythonProject/web_task/functions.py
new file mode 100644
--- /dev/null	(revision 43fff3bc4ba0a8efd83a1207464b347c2fc00ac6)
+++ b/PythonProject/web_task/functions.py	(revision 43fff3bc4ba0a8efd83a1207464b347c2fc00ac6)
@@ -0,0 +1,20 @@
+FILEPATH = "todos.txt"
+
+
+def read_todos(filepath=FILEPATH):
+    '''quando colocado na função, nao precisa repetir no código abaixo.
+    Só precisa alterar caso o txt tenha mudado de nome ou local armazenado.'''
+    with open(filepath, "r") as file_local:
+        todos_local = file_local.readlines()
+    return todos_local
+
+def write_todos(todos_arg, filepath=FILEPATH):
+    with open(filepath, "w") as file:
+        file.writelines(todos_arg)
+
+
+#caso queira que algo nao apareça no import
+#precisa ter uma condição if, por exemplo:
+
+if __name__ == "__main__":
+    print(read_todos())
\ No newline at end of file
